arm-containerregistry/2017-03-01/examples/RegistryListCredentials.json

```diff
@@ -0,0 +1,25 @@
+{
+    "parameters": {
+        "api-version": "2017-03-01",
+        "subscriptionId": "subid",
+        "resourceGroupName": "rgname",
+        "registryName": "registry"
+    },
+    "responses": {
+        "200": {
+            "body": {
+                "username": "registry",
```

Comment:
Yes, out service will generate a username. I probably can change the name so it can be less confusing.